pq-crypto/bike_r3/gf2x_inv.c

```diff
@@ -94,6 +104,10 @@ bike_static_assert((R_BITS == 24659), gf2x_inv_r_doesnt_match_parameters);
 // c = a^{-1} mod x^r-1
 void gf2x_mod_inv(OUT pad_r_t *c, IN const pad_r_t *a)
 {
+  // Initialize gf2x methods struct
+  gf2x_ctx ctx;
```

Comment:
You should zero initialize ctx: `gf2x_ctx ctx = {0};`

Reply:
Done